vendor/code-highlight-linenums.js

```diff
@@ -0,0 +1,96 @@
+(function(root) {
```

Comment:
Please add a comment like

```
Copied from https://github.com/OverZealous/code-highlight-linenums v0.2.1
MIT Licensed
```

ensuring of course that that's really the right URL and version number.